.github/actions/build/action.yml

```diff
@@ -67,7 +79,7 @@ runs:
   steps:
     - name: Set up additional environment variables
       env:
-        ADDITIONAL_VARIABLES: ${{ inputs.env_vars }}
+        ADDITIONAL_VARIABLES: ${{ inputs.env_vars }}  # Why is this used? On line 99 env_vars is used.
```

Comment:
This is used to pass any ENV variables that are more of a configuration, e.g. `APP_ENV`.